lang/java/reef-wake/wake/pom.xml

```diff
@@ -151,6 +167,15 @@ under the License.
             <groupId>net.jcip</groupId>
             <artifactId>jcip-annotations</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.avro</groupId>
+            <artifactId>avro</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.github.lukehutch</groupId>
+            <artifactId>fast-classpath-scanner</artifactId>
+            <version>LATEST</version>
```

Comment:
Shall we define a version the the parent pom for this? Also, we probably should snap to a fixed version for stability.